### ISSUE TITLE
Please make it build with ghc 9.10

### ISSUE THREAD

**Issue by erikd** [haskellari/postgresql-simple#139]
Currently need:
```
diff --git a/postgresql-simple.cabal b/postgresql-simple.cabal
index ca47515..dee677e 100644
--- a/postgresql-simple.cabal
+++ b/postgresql-simple.cabal
@@ -77,10 +77,10 @@ library
 
   -- GHC bundled libs
   build-depends:
-      base              >=4.12.0.0 && <4.20
+      base              >=4.12.0.0 && <4.21
     , bytestring        >=0.10.8.2 && <0.13
-    , containers        >=0.6.0.1  && <0.7
-    , template-haskell  >=2.14.0.0 && <2.22
+    , containers        >=0.6.0.1  && <0.8
+    , template-haskell  >=2.14.0.0 && <2.23
     , text              >=1.2.3.0  && <1.3  || >=2.0 && <2.2
     , time-compat       >=1.9.5    && <1.12
     , transformers      >=0.5.6.2  && <0.7
```

**Comment by phadej** [comment]
Blocked on `postgresql-libpq`, which in turn is blocked on not having `cabal-install` release.

**Comment by erikd** [comment]
What is the issue with `cabal-install` ?

**Comment by erikd** [comment]
```
> cabal outdated
Outdated dependencies:
base >=4.12.0.0 && <4.20 (latest: 4.20.0.1)
containers >=0.6.0.1 && <0.7 (latest: 0.7)
hashable >=1.4.3.0 && <1.5 (latest: 1.5.0.0)
template-haskell >=2.14.0.0 && <2.22 (latest: 2.22.0.0)
```

**Comment by erikd** [comment]
`postgresql-lipq` has been updated. What is the ETA for this?

**Comment by phadej** [comment]
I'm waiting for https://github.com/haskell-unordered-containers/unordered-containers/issues/492, in hope it will be bumped soon.

Based on #141 there are no code changes required, so please use `allow-newer` in the meantime.

**Comment by erikd** [comment]
I am already using `allow-newer` but my PR cannot be merged until the `allow-newer` is removed.

**Comment by andreasabel** [comment]
hashable-1.5 is not needed for GHC 9.10.
I don't think it makes sense to wait for the ecosystem to adopt hashable-1.5 before making libs available for GHC 9.10.

E.g. I have put hashable-1.5 on low prio for my projects.  I might take it with the GHC 9.12 alpha wave.  Others might decide in a similar fashion.

**Comment by phadej** [comment]
> hashable-1.5 is not needed for GHC 9.10.

as I said, I hope it will be bumped soon. As now is a summer time, I may assume `unordered-containers` maintainers are having their holidays.

---

It's not, but it will be for 9.12, as I have no intention to bump `hashable-1.4` support for 9.12 then. So either we are blocked now, or we will be blocked in 4-5 months again.
